### ISSUE TITLE
app connecting issue

### ISSUE THREAD

**Issue by blesseco** [proxidize/proxidize-android#66]
![image](https://github.com/proxidize/proxidize-android/assets/136700016/89abddc3-3ffd-442a-86c2-6e9fb3011d5a)
getting this restart issue even i restarted my phone many time 
can you help me fix it
and also please make video tutorial on custom server

**Comment by oloho** [comment]
Hi
Check this
https://github.com/proxidize/proxidize-android/issues/18

**Comment by AbedElez** [comment]
What does your server settings look like?

**Comment by aoliva31** [comment]
Hi, I'm facing the same issue, when I'm using a prepaid sim card, (it looks like that they banned the proxy protocol ?) however, I can connect to my custom server using normal sim card (Orange) or if I connect using the Wi-Fi it also works.

Here is my server config : 
![image](https://github.com/proxidize/proxidize-android/assets/46679017/be931415-8511-43e3-a7e0-b9bf6fca5a84)

Logs (traces only on Wi-Fi, when using 4G, I have no logs at all) : 
![image](https://github.com/proxidize/proxidize-android/assets/46679017/5ffb53f6-caef-46c8-bee5-85a6b5c5eb9c)

https://github.com/proxidize/proxidize-android/assets/46679017/2024c633-0d85-4306-9f27-811036049e33


Working on WIFI, but it's not on 4G (I have internet access with great connection)

Similar issues : 
- https://github.com/proxidize/proxidize-android/issues/59 (I tried the version before and it's not working)
- https://github.com/proxidize/proxidize-android/issues/16
- https://github.com/proxidize/proxidize-android/issues/66

Hope it helps !

**Comment by AbedElez** [comment]
Looks like your ISP/carrier might be doing DPI to block the proxy/tunnel. I'll take a closer look once I have the time.

**Comment by PetraChristalbert** [comment]
I created my own proxy with ubuntu, i open the port 51523, ACCEPT at all sudo iptables -S, the proxy details showed, and then time to use the proxy.
![image](https://github.com/proxidize/proxidize-android/assets/151901650/5326c5be-6538-47d2-b577-a50961a27347)

but the proxy didnt work at all... I am really confused, tried everything for 2 days... Hopefully there is something i need to fix...
![image](https://github.com/proxidize/proxidize-android/assets/151901650/54b5cfd2-a92a-4cda-b60c-2307c619d1d2)

Just for information, i am using Ubuntu VM from VULTR. I tried to use the default proxy from Proxidize and it totally works. but the proxies i made by deploying own server is just doesnt works at all...
